### DISCUSSION TITLE
Styling Dialog

### DISCUSSION THREAD

**Discussion by kacperbarski** [PrismLibrary/Prism#2359] [WPF]
Hello all,

Firstly, I'd like to say your Pluralsight course "Introduction to Prism for WPF" is fantastic Brian! I completed it recently and just have one question from the last module to which I can't find an answer online.

You mentioned in your course the style must target the Window type. This is where I am having issues because I am trying to implement a modal window that **is not** hosted inside a window.

This is what I have now with Prism:
![image](https://user-images.githubusercontent.com/79811961/109435095-97379600-7a10-11eb-9d7c-2d658574bef6.png)

And this is what I am working on (struggling to implement Prism Dialog to my user control):
![image](https://user-images.githubusercontent.com/79811961/109435231-568c4c80-7a11-11eb-93f1-8401c268d4ee.png)

As you can see from the above snapshot my dialog user control does not have a window. The main Grid inside my ShellWindow has an x:Name assigned and I use this to add my user control to it. I have no idea how I could achieve this with Prism. I tried implementing my own version of IDialogWindow as you have shown in this [Youtube](https://www.youtube.com/watch?v=XmR2ManyppU) video, but had no luck.

Any help is greatly appreciated

**Comment by brianlagunas** [comment]
This isn't really a Prism thing, it's an abstraction thing. It definitely is possible as I have done a dialog service like this myself in the past. However, this does fall outside the scope of my community support and into my paid services. If you'd like my consultation services just let me know. Paying a small fee now could save you thousands in the long run.
